### ISSUE TITLE
flags should be GObject.ParamFlags in GObject.ParamSpec.object()

### ISSUE THREAD

**Issue by spazziale** [gjsify/ts-for-gir#181]
Using  @girs/gobject-2.0: "^2.80.2-4.0.0-beta.9",

now the only type accepted is "string"

**Comment by JumpLink** [comment]
@spazziale sorry, my mistake. I hope this has been fixed. I release beta.10 now, can you test if it works again?

**Comment by spazziale** [comment]
seems to not be solved, maybe because i used
```
import GLib from '@girs/glib-2.0';
import GObject from '@girs/gobject-2.0';
```
due  to #182

**Comment by JumpLink** [comment]
@spazziale are you generating the types by yourself or do you use the types published on npm? If you generates the types by yourself then there is now a bracking change: `package.json` support is now disabled by default. But you can enable it again with `--packlage`.

**Comment by JumpLink** [comment]
@spazziale Maybe you are simply missing a
```ts
import '@girs/gjs';
import '@girs/gjs/dom';
``` 
?

**Comment by spazziale** [comment]
> @spazziale are you generating the types by yourself or do you use the types published on npm? If you generates the types by yourself then there is now a bracking change: `package.json` support is now disabled by default. But you can enable it again with `--packlage`.

no i'm using the ones from npm

**Comment by spazziale** [comment]
> ```ts
> import '@girs/gjs';
> import '@girs/gjs/dom';
> ```

nothing happen,
here is my [tsconfig.json](https://github.com/user-attachments/files/16325647/tsconfig.json) that was working with beta.5

**Comment by JumpLink** [comment]
Can you try `"moduleResolution": "NodeNext"`?

**Comment by spazziale** [comment]
> Can you try `"moduleResolution": "NodeNext"`?
using beta.11
import now works, should it be documented somewhere or it's something about my config ( still new to js/ts, sorry)?
sadly the original error is still not fixed.

also i noticed that GObject constructor() doesn't take in input constructProperties

```
constructor(constructProperties = {}) {
        super(constructProperties); // Expected 0 arguments, but got 1.
    }
```

thanks for your work

**Comment by JumpLink** [comment]
yes, we should document it. I think it's because we use ESM + Node.js for package resolution and that's what NodeNext is for. `“moduleResolution”: “bundler”` should also work, but then a bundler like `esbuild` is needed as we do this in the examples.
